cfme/tests/cli/test_evmserverd.py

```diff
@@ -23,28 +22,31 @@ def test_evmserverd_stop(ssh_client):
 
     Steps:
         * Remember all server names from ``service evmserverd status`` command.
+            * Or the bin/rake evm:status on 5.5+ since the systemd status does not show that, this
+                applies also for next references to status.
         * Issue a ``service evmserverd stop`` command.
-        * Periodically check output of ``service evmserverd stop`` that all the servers are stopped.
+        * Periodically check output of ``service evmserverd status`` that all servers are stopped.
+        * For 5.5+: Really call ``service evmserverd status`` and check that the mentions of
+            stopping the service are present.
     """
-    if current_version() < "5.5":
-        server_names = {server["Server Name"] for server in ssh_client.status["servers"]}
-        assert ssh_client.run_command("service evmserverd stop").rc == 0
+    server_names = {server["Server Name"] for server in ssh_client.status["servers"]}
+    assert ssh_client.run_command("service evmserverd stop").rc == 0
 
-        def _check():
-            status = {server["Server Name"]: server for server in ssh_client.status["servers"]}
-            for server_name in server_names:
-                if status[server_name]["Status"] != "stopped":
-                    return False
-            return True
+    @pytest.wait_for(timeout="2m", delay=5)
+    def servers_stopped():
+        status = {server["Server Name"]: server for server in ssh_client.status["servers"]}
+        for server_name in server_names:
+            if status[server_name]["Status"] != "stopped":
+                return False
+        return True
 
-        wait_for(_check, num_sec=120, delay=5, message="servers stopped")
-    else:
-        assert ssh_client.run_command("service evmserverd stop").rc == 0
+    if current_version() >= "5.5":
         status = ssh_client.run_command("service evmserverd status")
         assert "Stopped EVM server daemon" in status.output
         assert "code=exited" in status.output
 
 
+@pytest.mark.uncollectif(lambda: current_version() >= "5.5")
```

Comment:
Just curious - why do we want to skip this one in `5.5+` now? Is it because running `service evmserverd start` now gives us no output / no rc that would tell us that evmserverd is already running?